experiments/start_serial_experiment_examin.py

```diff
@@ -0,0 +1,69 @@
+import subprocess
+import re
+import argparse
+from subprocess import Popen, PIPE
+from stats import save_stats, compute_stats
+
+def createParser ():
```

Comment:
Please preserve the naming convention from the other scripts: use dash as a separator and lower case letters only for functions and variables